profiles/common/modules/features/nexteuropa_trackedchanges/nexteuropa_trackedchanges.module

```diff
@@ -0,0 +1,736 @@
+    'entity views field' => TRUE,
+    'computed' => TRUE,
+  );
+  $info['netc_info']['properties']['rel_entity_language'] = array(
+    'label' => t('Concerned entity language'),
+    'description' => t('Concerned entity language where tracked changes have been detected.'),
+    'type' => 'text',
+    'schema field' => 'rel_entity_language',
+    'getter callback' => 'entity_property_verbatim_get',
+    'setter callback' => 'entity_property_verbatim_set',
+    'entity views field' => TRUE,
+  );
+  $info['netc_info']['properties']['rel_entity_path'] = array(
+    'label' => t('Concerned entity path'),
+    'description' => t('Concerned entity path where tracked changes have been detected.'),
+    'type' => 'text',
+    'schema field' => 'rel_entity_path',
+    'getter callback' => 'entity_property_verbatim_get',
+    'setter callback' => 'entity_property_verbatim_set',
+    'entity views field' => TRUE,
+  );
+  $info['netc_info']['properties']['scanned'] = array(
+    'label' => t('Scanning date of the info'),
+    'description' => t('Timestamp of the date when the info entity has been created.'),
+    'type' => 'date',
+    'schema field' => 'scanned',
+    'getter callback' => 'entity_property_verbatim_date_get',
+    'setter callback' => 'entity_property_verbatim_set',
+    'entity views field' => TRUE,
+  );
+}
+
+/**
+ * Implements hook_views_api().
+ *
+ * @see nexteuropa_trackedchanges_views_default_views()
+ */
+function nexteuropa_trackedchanges_views_api() {
+  return array(
+    'api' => 3.0,
+    'path' => drupal_get_path('module', 'nexteuropa_trackedchanges') . '/views',
+  );
+}
+/**
+ * Implements hook_views_pre_render().
+ *
+ * @see nexteuropa_trackedchanges_views_default_views()
+ */
+function nexteuropa_trackedchanges_views_pre_render(&$view) {
+  // Alter the display of the language list shown with each row on the
+  // "Content tracked changes" page.
+  $implied_display = array(
+    'default',
+    'page',
+  );
+  if (($view->name == 'content_tracked_changes') && in_array($view->current_display, $implied_display)) {
+    $results = $view->result;
+
+    if ($results) {
+      $language_list = language_list();
+      foreach ($results as $key => $result) {
+        $entity_path = '';
+        $entity_moderation_state = '';
+        if (isset($result->_field_data)) {
+          $nect_info_entity = $result->_field_data['info_id']['entity'];
+          $nect_info_entity_type = $result->_field_data['info_id']['entity_type'];
+          $wrapper = entity_metadata_wrapper($nect_info_entity_type, $nect_info_entity);
+          $entity_path = $wrapper->rel_entity_path->value();
+          $entity_moderation_state = $wrapper->rel_entity_state->value();
+        }
+
+        if ($entity_path) {
+          $languages_field = $result->field_field_netc_info_languages;
+          if ($languages_field) {
+            foreach ($languages_field as $key2 => $lang_value) {
+              $raw = $lang_value['raw']['value'];
```

Comment:
I was wondering if using the full language name wasn't more appropriate, but as you said, if there's content with more than 20 translations, it might break the UI. Or we could use a list ? Dunno.